src/Microsoft.ML.Transforms/Text/doc.xml

```diff
@@ -0,0 +1,165 @@
+        Features are made of (word/character) n-grams/skip-grams​ and the number of features are equal to the vocabulary size found by analyzing the data.
+      </remarks>
+      <example>
+        <code>
+          pipeline.Add(new TextFeaturizer(&quot;Features&quot;, &quot;SentimentText&quot;)
+          {
+            KeepDiacritics = false,
+            KeepPunctuations = false,
+            TextCase = TextNormalizerTransformCaseNormalizationMode.Lower,
+            OutputTokens = true,
+            StopWordsRemover = new PredefinedStopWordsRemover(),
+            VectorNormalizer = TextTransformTextNormKind.L2,
+            CharFeatureExtractor = new NGramNgramExtractor() { NgramLength = 3, AllLengths = false },
+            WordFeatureExtractor = new NGramNgramExtractor() { NgramLength = 2, AllLengths = true }
+          });
+        </code>
+      </example>
+    </member>
+
+    <member name="WordTokenizer">
+      <summary>
+        The input to this transform is text, and the output is a vector of text containing the words (tokens) in the original text. 
+        The separator is space, but can be specified as any other character (or multiple characters) if needed.
+      </summary>
+      <remarks>
+        The input for this transform is a <see cref="Microsoft.ML.Runtime.Data.DvText">DvText</see> or a vector of <see cref="Microsoft.ML.Runtime.Data.DvText">DvTexts</see>,
+        and its output is a vector of DvTexts, corresponding to the tokens in the input text.
+        The output is generated by splitting the input text, using a set of user specified separator characters.
+        Empty strings and strings containing only spaces are dropped.
+        This transform is not typically used on its own, but it is one of the transforms composing the Text Featurizer.
+      </remarks>
+      <example>
+        <code>
+          pipeline.Add( new WordTokenizer(&quot;TextColumn&quot;){ TermSeparators = &quot;&apos; &apos;, &apos;\t&apos;, &apos;;&apos;&quot;  } );
+        </code>
+      </example>
+    </member>
+
+    <member name="NgramTranslator">
+      <summary>
+        This transform produces a bag of counts of n-grams (sequences of consecutive values of length 1-n) in a given vector of keys. 
+        It does so by building a dictionary of n-grams and using the id in the dictionary as the index in the bag.
+      </summary>
+      <remarks>
+        This transform produces a matrix of token ngrams/skip-grams counts for a given corpus of text.
+        The n-grams are represented as count vectors, with vector slots corresponding to n-grams.
+        Embedding ngrams in a vector space allows their contents to be compared in an efficient manner. 
+        The slot values in the vector can be weighted by the following factors:
+        <list>
+          <item><description>term frequency - The number of occurrences of the slot in the text</description></item>
```

Comment:
:bulb:

```xml
<item>
  <term>term frequency</term>
  <description>The number of occurrences of the slot in the text</description>
</item>
```

Reply:
this is marked as resolved but I don't see the change?

---
In reply to: [202777588](https://github.com/dotnet/machinelearning/pull/529#discussion_r202777588) [](ancestors = 202777588)

Reply:
it is resolved. Compare iteration 15 with 11. 

Used to be:
<item>
            <description>term frequency - The number of occurrences of the slot in the text</description>
</item>

got changed to:
<item>term frequency
            <description> the number of occurrences of the slot in the text</description>
          </item>

notice the location of  "term frequency"

---
In reply to: [203202981](https://github.com/dotnet/machinelearning/pull/529#discussion_r203202981) [](ancestors = 203202981,202777588)

Reply:
I mean there are no `<term>` tags. Is there a reason why? If it's possible, I'd rather have them :)

---
In reply to: [203204792](https://github.com/dotnet/machinelearning/pull/529#discussion_r203204792) [](ancestors = 203204792,203202981,202777588)

Reply:
good catch. 

---
In reply to: [203205760](https://github.com/dotnet/machinelearning/pull/529#discussion_r203205760) [](ancestors = 203205760,203204792,203202981,202777588)